vms/registration/views.py

```diff
@@ -186,11 +193,22 @@ def post(self, request):
 
                     volunteer.reminder_days = 1
                     volunteer.save()
-                    registered = True
-
-                    messages.success(request,
-                                     'You have successfully registered!')
-                    return HttpResponseRedirect(reverse('home:index'))
+                    user.is_active = False
```

Comment:
do you need a user.save after this ? If yes, please make sure this view do not call user.save() more than once.

Reply:
No I don't need it.